examples/vmod_object/src/lib.rs

```diff
@@ -17,7 +17,7 @@ const EMPTY_STRING: String = String::new();
 
 // kv only contains one element: a mutex wrapping a String->String hashmap
 pub struct kv {
-    mutexed_hash_map: Mutex<HashMap<String, String>>,
+    data: Mutex<HashMap<String, String>>,
```

Comment:
since it's an example, I'd rather keep the name, just to make it painfully clear what that thing is to newcomers